src/annotations/AnnotationManager.php

```diff
@@ -299,6 +306,22 @@ protected function getAnnotations($class_name, $member_type = self::MEMBER_CLASS
                 }
             }
 
+            if ($this->_traitsSupported && $member_name) {
+                $traitAnnotations = array();
+
+                if (isset($file->traitMethodOverrides[$class_name][$member_name])) {
+                    list($traitName, $originalMemberName) = $file->traitMethodOverrides[$class_name][$member_name];
+                    $traitAnnotations = $this->getAnnotations($traitName, $member_type, $originalMemberName);
+                } else {
+                    foreach ($reflection->getTraitNames() as $traitName)
+                    {
+                        $traitAnnotations += $this->getAnnotations($traitName, $member_type, $member_name);
+                    }
+                }
+
+                $annotations = array_merge($traitAnnotations, $annotations);
```

Comment:
I have a test case for this, actually. https://github.com/benesch/php-annotations/blob/trait-docblocks/test/suite/Annotations.test.php#L629-L632

From the docs: 

> An inherited member from a base class is overridden by a member inserted by a Trait. The precedence order is that members from the current class override Trait methods, which in turn override inherited methods.

So I figured we should inherit annotations exactly the same way. Would you prefer I change that logic?